day_6/exercises/ex03_exercises.rb

```diff
@@ -0,0 +1,90 @@
+class MyCar
+  attr_accessor :color, :model, :speed
+  attr_reader :year
+
+  def initialize(y, c, m, s)
```

Comment:
Be sure to always be verbose and clear when naming parameters or variables. One letter is definitely not either. Keep it top of mind moving forward!